Gemfile

```diff
@@ -25,7 +25,12 @@ gem "aws-sdk-s3", require: false
 gem "image_processing", ">= 1.2"
 gem "active_storage_validations"
 
+# API
 gem "jbuilder" # JSON templating
+gem "versioncake"
+gem "jbuilder_pagination_plus", require: "jbuilder/pagination" # Depends on Kaminari and Jbuilder
+
+gem "kaminari"
```

Comment:
Thoughts on using [pagy](https://github.com/ddnexus/pagy) instead? Aside from the reasons on the README, it'll also negate the need for `jbuilder_pagination_plus`, which doesn't look widely used or maintained anyway.

Reply:
I remember we had tried it for https://github.com/hackclub/bank/ but found no performance improvement. Nevertheless, let's give it a shot!

[`611f0b3` (#57)](https://github.com/hackclub/hackathons-backend/pull/57/commits/611f0b304d83b6e15c06f72d481a9008d369a5d7)